config/sidekiq.yml

```diff
@@ -10,3 +10,9 @@
   email_generation:
     every: '5s'
     class: EmailGenerationWorker
+  daily_digest_initiator:
+    cron: '30 8 * * *'
+    class: DailyDigestInitiatorWorker
+  weekly_digest_initiator:
+    cron: '30 8 * * 6'
+    class: WeeklyDigestInitiatorWorker
```

Comment:
Might be useful to just have a comment about what these translate to in human terms as 6th day of week isn't too fast to grok. 

Also not really a discussion for here, but why'd we go for Saturday over Monday?